apps/site/assets/ts/schedule/components/schedule-finder/ScheduleModalContent.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { ReactElement, useReducer, useEffect } from "react";
+import { SelectedDirection, SelectedOrigin } from "../ScheduleFinder";
+import UpcomingDepartures from "./UpcomingDepartures";
+import { BareRoute as Route, RouteType } from "../../../__v3api";
+import { SimpleStop, StopPrediction } from "../__schedule";
+import isSilverLine from "../../../helpers/silver-line";
+
+const stopInfo = (
+  selectedOrigin: string,
+  stops: SimpleStop[]
+): SimpleStop | undefined => stops.find(({ id }) => id === selectedOrigin);
+
+const stopNameLink = (
```

Comment:
I had converted some of these functions to their own small components in previous PR -- sure you don't want to keep those?

Reply:
We don't that in other places and I don't think it's necessary.